examples/lusid/quotes/upload_fx_quotes.sql

```diff
@@ -0,0 +1,33 @@
+-- =====================================================
+
+-- Description:
+-- In this query, we run an ETL process on FX quotes
+
+-- =====================================================
+
+-- Extract FX data from LUSID Drive
+
+@fx_price_data =
+use Drive.Excel
+--file=/luminesce-examples/daily_quotes.xlsx
+--worksheet=fx
+enduse;
+
+-- Run transformation on the data
+
+@quotes_for_upload = select
+'CurrencyPair' as InstrumentIdType,
+ccy_pair as Instrumentid,
+'luminesce-examples' as QuoteScope,
+'Rate' as QuoteType,
+'Lusid' as Provider,
+'Mid' as Field,
+price_date as QuoteEffectiveAt,

```

Comment:
For reference this will need to be the same as the valuation date or within the limits set in the recipe for it :)